components/Cards/ResourceCard/ResourceCard.js

```diff
@@ -62,15 +62,18 @@ function ResourceCard({
 }) {
   const [upVotes, setUpVotes] = useState(upvotes);
   const [downVotes, setDownVotes] = useState(downvotes);
-  const didUpvote = userVote === possibleUserVotes.upvote;
-  const didDownvote = userVote === possibleUserVotes.downvote;
+  const [userVoteDirection, setUserVoteDirection] = useState(userVote);
```

Comment:
This line shouldn't be necessary. Keep in mind, `userVote` is a prop because it's not just state for the current session, but their vote direction also needs to be provided by the back-end. In other words, if you upvote a resource, close your browser, re-open the browser, and go back to the same resource - it should stills how that resource as upvoted.

The state for a ResourceCard's userVote should be managed via the resource page file for that reason and it's initial state should be derived from the request for data on the resources.